docs/src/pages/components/radiogroupfield/examples/DirectionExample.tsx

```diff
@@ -0,0 +1,27 @@
+import { Radio, RadioGroupField } from '@aws-amplify/ui-react';
+
+export const DirectionExample = () => (
+  <>
+    <RadioGroupField
+      label="Language"
+      name="example-7"
+      defaultValue="html"
+      labelHidden
+    >
+      <Radio value="html">html</Radio>
+      <Radio value="css">css</Radio>
+      <Radio value="javascript">javascript</Radio>
+    </RadioGroupField>
+    <RadioGroupField
+      label="Language"
+      name="example-8"
+      defaultValue="html"
+      direction="row"
+      labelHidden
+    >
+      <Radio value="html">html</Radio>
+      <Radio value="css">css</Radio>
+      <Radio value="javascript">javascript</Radio>
+    </RadioGroupField>
```

Comment:
I noticed that we repeatedly using this pattern in this PR, should we consider making a component?

Reply:
The code will be displayed on docs side as examples. We probably don't want to encapsulate that

Reply:
Yup anything in an examples folder on the docs site should be stand-alone and be easily readable for users as we display this code in a code block.

Reply:
oh, I see. I didn't notice those are examples